drivers/wd/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="onitu-webdav",
+    version="0.1",
+    url="http://onitu.github.io",
+    description="Store files on a remote machine using webdav through onitu.",
+    license="MIT",
+    packages=find_packages(),
+    install_requires=["pycurl", "lxml", "webdavclient"],
+    package_data={'': ['manifest.json']},
+    entry_points={
+        'onitu.drivers': [
+            'wd = onitu_webdav'
```

Comment:
Because I have a name collision when I tried to import webdav in my code. I change my webdav to wd in all the places. I can try to change to webdav.